.env.secrets.example

```
@@ -0,0 +1,16 @@
+# Cloudflare Worker Secrets
+# Copy this file to .env.secrets and fill in your actual secret values
+# This file is gitignored for security
+
+# GitHub API Token for repository access
+GITHUB_TOKEN="your_github_token_here"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)